build.gradle

```diff
@@ -0,0 +1,47 @@
+
+plugins {
+    id 'java-gradle-plugin'
+    id 'com.gradle.plugin-publish' version '0.10.1' 
+    id 'maven-publish'
+}
+
+repositories {
+    jcenter()
```

Comment:
the gradle plugin docs recommend using jcenter() as the repository, as that's what gradle will use by default at runtime to resolve plugins.  It's better for downstream users if jcenter() 'just works' without having to add a bunch of buildscript repositories, so better to build your plugin a jcenter() to make sure there aren't unexpected issues that can't be resolved during development.